### DISCUSSION TITLE
What type of game is PE2D best suited for?

### DISCUSSION THREAD

**Discussion by plurifyincorporated** [PhaserEditor2D/PhaserEditor2D-v3#96] [Q&A] [answered]
I'm new to Phaser Editor 2D but super excited about the potential.  Can anyone give some idea of what the editor (in its current iteration)  is best suited for?  
- side stroller
- tiled rpg 
- etc
Are there good examples of different types of games that have been built with the platform and some limitations that are either being worked on or not planned to be addressed?

**Comment by PhaserEditor2D** [comment, marked as answer]
Hi!

You can check the kind of projects the users are making with Phaser Editor here: 

https://phasereditor2d.com/blog/category/made-with-pe

https://phasereditor2d.com/blog/category/tutorials

However, you can do whatever kind of game with the editor. If there is a specific feature that is not available, you can solve it using any other tool (like for example, creating a tilemap level with Tiled). Phaser Editor allows using third-party tools or adding code manually.

I recommend you check the docs: https://help.phasereditor2d.com/v3
